### ISSUE TITLE
Functional Tests with config change

### ISSUE THREAD

**Issue by purnimavenkatasubbu** [atsign-foundation/at_server#542]
**Is your feature request related to a problem? Please describe.**
Add tests for the features that need a server config change.

- [x]  Connection limit test
- [x] Assert behavior of compaction and other scheduled or triggered jobs  - Compaction Stats
- [ ] Timeout tests

This is equivalent to #505

**Comment by purnimavenkatasubbu** [comment]
Currently, the timeouts in the server are hardcoded in the code for a default value. Hence the functional tests cannot be written around these at the moment.

**Comment by gkc** [comment]
We need ability - in test environments **only** initially - to dynamically adjust server config from an authenticated client. Filed https://github.com/atsign-foundation/at_server/issues/588

**Comment by purnimavenkatasubbu** [comment]
The tests for connection limit, compaction, and access logs are pushed to the branch [issue#542](https://github.com/atsign-foundation/at_server/tree/issue%23542).

This needs a config change in the server for the tests to pass.

**Comment by purnimavenkatasubbu** [comment]
@gkc I need a secondary config change (for instance inbound_connection_limit set to 2, compactionJobRun set to 1minute) for the tests to pass in the GitHub Actions.
Can we pass a sed command in the GitHub actions before running the tests that need a config change? 
sed 's/inbound_max_limit: 50/inbound_max_limit: 1/' col | tee config/config.yaml

Or is there any better way to do this?

**Comment by purnimavenkatasubbu** [comment]
This task is dependent on #588 to adjust the config changes for the test atsigns. Hence no progress was made in this sprint.

**Comment by murali-shris** [comment]
moving this to backlog till #588 is done

**Comment by gkc** [comment]
@purnimavenkatasubbu Can you specify precisely which parameters need to be dynamically configurable right now please? Is it just inbound_connection_limit and compactionJobRun?

**Comment by purnimavenkatasubbu** [comment]
Yes, @gkc. I need to set inbound_connection_limit and compactionJobRun before the test run and reset it to the default value after the test completion.

**Comment by ksanty** [comment]
@purnimavenkatasubbu should this be moved to high priority, so we can work and close in PR43/44?

**Comment by gkc** [comment]
It's just about done, it's been worked On steadily for the past few weeks

**Comment by ksanty** [comment]
> It's just about done, it's been worked On steadily for the past few weeks

Should it be in PR42 as unplanned work?

**Comment by gkc** [comment]
Maybe? Purnima have you been active on this in the current sprint?

**Comment by gkc** [comment]
Triaged in an old-issue sweep. This was blocked on #588 (dynamic config adjustment), delivered and closed 2022-06-27: the `config` verb's set/reset/print (gated behind server testing mode) now exists, and the exact parameters named here (inbound connection limit, compaction frequency) are `ModifiableConfigs` members driven from the functional suite. The issue was simply never closed after its dependency landed. Closing as completed.
